src/app/components/shared/website-capability-warning/website-status-warning.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { WebsiteStatusService } from "@baw-api/website-status/website-status.service";
+import { websiteStatusMenuItem } from "@components/website-status/website-status.menu";
+import { withUnsubscribe } from "@helpers/unsubscribe/unsubscribe";
+import { WebsiteStatus } from "@models/WebsiteStatus";
+import { takeUntil } from "rxjs";
+
+@Component({
+  selector: "baw-website-status-warning",
+  template: `
+    <div *ngIf="!isCapabilityHealthy" class="alert alert-danger">
+      The ability to {{ websiteCapabilityText }} is temporarily unavailable.
+      Please try again later.
+
+      <div>
+        <a [strongRoute]="websiteStatusRoute">More Information</a>
+      </div>
+    </div>
+  `,
+})
+export class WebsiteStatusWarningComponent
+  extends withUnsubscribe()
+  implements OnInit
+{
+  public constructor(private api: WebsiteStatusService) {
+    super();
+  }
+
+  @Input({ required: true }) public feature: keyof WebsiteStatus;
```

Comment:
I'm not wholly convinced this is needed.

Anywhere where it matters enough to put this warning, any service not being available is enough for it not to work. 

If disk, or database are down then uploads, downloads, and everything is non-functional. 

Then again, testing for the status of the upload server is relevant to only the harvest pages. 

Ok I argued myself in favour of this as it is, as you were